2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,10 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let temparatureStatements  = [];
```

Comment:
Typo here! Typos in variable names often cause bugs so it's really worth picking them up. There is a spellchecker package you can install in VSCode  https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker